frontend/src/components/Election/Voting/GenericBallotView.js

```diff
@@ -0,0 +1,221 @@
+            {candidate.candidate_name}
+          </Typography>
+        </Grid>
+        <Choices
+          key={`starChoices${rowIndex}`}
+          rowIndex={rowIndex}
+          score={score}
+          onClick={onClick}
+          columns={columns}
+        />
+
+      </Grid>
+      {expanded && hasExpandedData &&
+      <>
+        <Grid container style={{ backgroundColor: rowColor }}>
+          <Grid item xs={1}>
+          </Grid>
+          <Grid item xs={5}>
+            {candidate.full_name &&
+            <Typography align='left' component="h6">
+              {candidate.full_name}
+            </Typography>}
+            {candidate.candidate_url && <Link color="inherit" href={candidate.candidate_url} target="_blank" underline="always">
+              <Typography align='left' component="h6">
+                Candidate URL
+              </Typography>
+            </Link>}
+            {candidate.party &&
+              <Typography align='left' s component="h6">
+                {`Party: ${candidate.party}`}
+              </Typography>}
+            {candidate.partyUrl &&
+              <Link color="inherit" href={candidate.partyUrl} target="_blank" underline="always">
+                <Typography align='left' component="h6">
+                  Party URL
+                </Typography>
+              </Link>}
+
+
+          </Grid>
+          {candidate.photo_filename &&
+          <Grid item xs={6}>
+            <img src={candidate.photo_filename} style={{ width: 200, height: 200 }} />
+          </Grid>}
+          </Grid>
+          
+          <Grid container style={{ backgroundColor: rowColor }}>
+          <Grid item xs={1}>
+          </Grid>
+          <Grid item xs={8}>
+            <Typography align='left' s component="p" style={{whiteSpace: 'pre-line'}}>
+              {candidate.bio}
+            </Typography>
+          </Grid>
+        </Grid>
+        </>
+      }
+
+    </>
+  )
+};
+
+// Represents the list of rows corresponding to the list of candidates
+const Rows = ({ candidates, scores, onClick, columns }) =>
+  candidates.map((row, n) => (
+    <>
+      <Row
+        rowIndex={n}
+        key={`starRow${n}`}
+        candidate={row}
+        party={row.party}
+        score={scores[n]}
+        onClick={(score) => onClick(n, score)}
+        columns={columns}
+      />
+      <Divider style={{ width: '100%' }} />
+    </>
+  ));
+
+// Represents the list of column headings for all possible scores
+const ColumnHeadings = ({columns}) => (
+  <>
+    <Grid container alignItems="stretch">
+      <Grid item xs={5}></Grid>
+      <ScoreColumnHeadings columns={columns}/>
+    </Grid>
+  </>
+);
+
+const ScoreColumnHeadings = ({columns}) =>
+  columns.map((columnValue, n) => (
+    <Grid item xs={1}>
+      <Typography align='center' variant="h6" component="h6">
+        {columns.length == 1? ' ' : columnValue}
+      </Typography>
+    </Grid>
+  ));
+
+
+
+// Renders a complete RCV ballot for a single race
```

Comment:
nit: remove rcv references in comments in this file and in other ballot views. Not sure why it was there in the first place.

Reply:
Sounds good, will address this in the next PR as well